lib/stages/hdf5FileWrite.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef HDF5_FILE_WRITE_H
+#define HDF5_FILE_WRITE_H
+
+#include "Config.hpp"
+#include "Stage.hpp" // for Stage
+#include "bufferContainer.hpp"
+
+#include <stdint.h> // for uint32_t
+#include <string>   // for string
+
+/**
+ * @class hdf5FileWrite
+ * @brief Stream a buffer to disk.
```

Comment:
Given this stage in limited to writing just one file, I'm tempted to either move it to `lib/testing/`, and/or make it clear in the comments this stage is used to run tests, and not to be used in a production pipeline (where we'd have to cycle file names).

Reply:
I added a comment.